templates/ingress.yaml

```diff
@@ -67,5 +68,24 @@ spec:
             pathType: Exact
             {{- end }}
           {{- end }}
+    - host: {{ printf "grafana.%s" .host | quote }}
```

Comment:
we shouldn't be setting up the ingress for Grafana, it's better to use their Values.ingress instead of having to maintain a copy of it

Reply:
good point, I'll work on that